class.jetpack.php

```diff
@@ -1566,7 +1566,7 @@ public function check_twitter_tags() {
 	 * @param array   $args         See definitions above
 	 */
 	public static function submit_security_report( $type = '', $plugin_file = '', $args = array() ) {
-		_deprecated_function( __FUNCTION__, 'always', 'Security reports feature has been removed' );
+		_deprecated_function( __FUNCTION__, 'jetpack-4.2', 'Security reports feature has been removed' );
```

Comment:
The third arg is for a replacement function to use, not an explanation. If there's not a replacement function, `null`.